about.html

```diff
@@ -0,0 +1,136 @@
+                <h1>
+                    Artificial intelligence,<br>are we talking?
+                </h1>
+                <p>
+                    Lorem ipsum dolor sit amet consectetur, adipisicing.<br>
+                    Animi numquam ipsa quis deleniti quia laudantium voluptatem perferendis, saepe cum aliquid optio unde!<br>A saepe pariatur commodi et
+                    blanditiis,<br>
+                    neque nemo, neque nemo!
+                    Lorem ipsum dolor sit amet consectetur, adipisicing elit. Animi numquam ipsa quis deleniti quia<br>
+                    laudantium voluptatem perferendis, saepe cum aliquid optio unde! A saepe pariatur commodi et
+                    blanditiis,<br>
+                    neque nemo, neque nemo!<br>
+                    laudantium voluptatem perferendis, saepe cum aliquid optio unde! A saepe pariatur commodi et
+                </p>
+                <p>Please contact us per Email for any further question<br>
+                    about Haiti Tech Summit 2023<br>
+                    <a mailto='htsummit2023@hts.org'>htsummit2023@hts.org</a></p>
+            </div>
+        </section>
+
+        <section class="program">
+            <h2>Haiti tech summit 2023 Logo</h2>
+            <hr>
+            <p>The logo of Haiti Tech Summit 2023 was decided through the logo competition from 8, June to 7, July.</p>
+            <img src="./public/images/logo.jpg" alt="Presentation of the logo of HTS">
+        </section>
+
+        <section class="speakers">
+            <h2>See the past Haiti tech summits</h2>
+            <hr>
+            <p>Take a look at the last two Haiti tech summit which took place in Haiti.</p>
+            <div class="cards">
+                <div class="card">
+
+                    <div class="front">
+                        <h2>2020</h2>
+                        <p>Haiti tech summit 2020</p>
+                    </div>
+                </div>
+                <div class="card">
+
+                    <div class="front">
+                        <h2>2022</h2>
+                        <p>Haiti tech summit 2022</p>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <section class="partner">
+            <h2>Partner</h2>
+            <hr>
+            <div class="cards">
+                <img src="./public/images/logo_airbnb.png" alt="Logo Airbnb">
+                <img src="./public/images/logo_nestle.png" alt="Logo Nestle">
+                <img src="./public/images/logo_mozilla.png" alt="Logo Mozilla">
+                <img src="./public/images/logo_blender.png" alt="Logo Bendler">
+            </div>
+        </section>
+
+        <section class="footer">
+            <div class="cards">
+                <img src="./public/images/creative_commons.png" alt="Logo Creative Commons black">
+                <img src="./public/images/creative_commons_white.png" alt="Logo Creative Commons white">
+                <div>
```

Comment:
- [ ] I would suggest you customize the size of the logo in the footer as was mentioned by the previous reviewer. it looks too wide.

- 
![Capture](https://user-images.githubusercontent.com/63972670/234877751-fc05fea4-933b-45e9-a5bf-d2aa09c0932f.JPG)
- Original Design
![Capture](https://user-images.githubusercontent.com/63972670/234878260-8e33115d-6c37-4c0b-82fc-56c65b75d048.JPG)
NB: I am requesting this because the project requirement recommends the original design.